server/src/test/java/org/elasticsearch/client/documentation/IndicesDocumentationIT.java

```diff
@@ -48,16 +54,14 @@ public void testPutMappingDocumentation() throws Exception {
         Client client = client();
 
         // tag::index-with-mapping
-        client.admin().indices().prepareCreate("twitter")   // <1>
-                .addMapping("\"tweet\": {\n" +              // <2>
-                        "  \"properties\": {\n" +
-                        "    \"message\": {\n" +
-                        "      \"type\": \"text\"\n" +
-                        "    }\n" +
-                        "  }\n" +
-                        "}")
+        client.admin().indices().prepareCreate("twitter")    // <1>
+                .addMapping("tweet", "message", "type=text") // <2>
```

Comment:
I wasn't sure which of the three "addMapping" variants to use in the client API docs, but went with this one since I think its the simplest. Both alternatives `addMapping(String type, Map<String, Object> source)` and `addMapping(String type, XContentBuilder source)` would require to setup a builder or a map with the mapping first, which would make this example quiet bloated IMHO.